pom.xml

```diff
@@ -126,7 +126,6 @@
 
     <commons.module.name>org.apache.commons.graph</commons.module.name>
 
-    <commons.release.version>1.0-beta1</commons.release.version>
```

Comment:
Hi @ppkarwasz 
I think we should keep this one line, it doesn't define a plug-in version. I makes some RM tasks easier.

Reply:
Fixed in https://github.com/apache/commons-graph/pull/30/commits/7ba1ec9490312128d201b85eaa856ee919f0942c